### PR TITLE
Feature/scroll wrapper into view

### PR DESCRIPTION
**Pull request by jakobe**
This PR closes # (reference issue number here)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] cookbook application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
N/A

Issue Number: N/A

## What is the new behavior?
Optionally allow wrapping `kirby-form-field` in a container with a `scroll-into-view` attribute to ensure wrapping element is scrolled into view on native keyboard (e.g. if having a background image behind an input/form field)

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: apps/cookbook/src/app/examples/examples.component.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ export class ExamplesComponent {
       const ionKeyboardDidShowEvent = new CustomEvent('ionKeyboardDidShow', {
         detail: { keyboardHeight: this.keyboardHeight },
       });
-      this.window.dispatchEvent(ionKeyboardDidShowEvent);
+      setTimeout(() => this.window.dispatchEvent(ionKeyboardDidShowEvent), 100);
```
Comment (mvindahl) [inline apps/cookbook/src/app/examples/examples.component.ts:71]: Lidt et magic number .. kunne måske være en konstant med et forklarende navn?

File: libs/designsystem/src/lib/components/form-field/form-field.component.ts [hunk 2]
```diff
@@ -39,7 +41,7 @@ export class FormFieldComponent implements AfterContentChecked, OnDestroy {
       this.isRegistered = true;
       document.dispatchEvent(
         new CustomEvent('ionInputDidLoad', {
```
Comment (mvindahl) [inline libs/designsystem/src/lib/components/form-field/form-field.component.ts:43]: Evt. en kommentar om, hvad det er, disse events sparker i gang? Måske med en reference til input-shims-filen.

**Review by mvindahl — APPROVED** [review]
Solved

File: libs/designsystem/src/lib/components/form-field/form-field.component.ts (hunk 2 above)
Comment (jakobe) [reply libs/designsystem/src/lib/components/form-field/form-field.component.ts:47]: Added 👍

File: apps/cookbook/src/app/examples/examples.component.ts (hunk 1 above)
Comment (jakobe) [reply apps/cookbook/src/app/examples/examples.component.ts]: Named magic number 👍

**Review by mvindahl — APPROVED** [review]

**Pull request merged**